spec/puck/dependency_resolver_spec.rb

```diff
@@ -105,6 +105,27 @@ module Puck
         end
 
         it 'does not write a bundle configuration' do
+          resolved_gem_dependencies # force evaluation
+          bundler_config = Pathname.new(app_dir_path).join('.bundle', 'config')
+          bundler_config.should_not exist
+        end
+      end
+
+      context 'with custom app_dir' do
```

Comment:
I'd reverse the tests. I.e. make testing with an explicit `app_dir` the default, test what happens otherwise in this context.